receiver/githubreceiver/README.md

```diff
@@ -70,12 +70,12 @@ receivers:
         collection_interval: 60s
         scrapers:
             scraper:
-                metrics:
+                metrics: #Optional
                     vcs.contributor.count:
                         enabled: true
-                github_org: myfancyorg
-                search_query: "org:myfancyorg topic:o11yalltheway" #Recommended optional query override, defaults to "{org,user}:<github_org>"
-                endpoint: "https://selfmanagedenterpriseserver.com"
+                github_org: <myfancyorg> 
+                search_query: "org:<myfancyorg> topic:<o11yalltheway>" #Recommended optional query override, defaults to "{org,user}:<github_org>"
```

Comment:
```suggestion
                search_query: "org:<myfancyorg> topic:<o11yalltheway>" # Recommended optional query override, defaults to "{org,user}:<github_org>"
```
Nit: I know this wasn't  from your change, but we may as well fix it while we're here.